packages/gatsby-plugin-netlify/README.md

```diff
@@ -1,3 +0,0 @@
-# gatsby-plugin-netlify
-
-Hey there 👋, gatsby-plugin-netlify has a new home. Go to https://github.com/netlify/gatsby-plugin-netlify to find the latest source code.
```

Comment:
We sure we won't to move out this "redirect"?

Reply:
It's been a year, I think it's safe to do that